### ISSUE TITLE
Suggestion: Turn off/on on specific websites

### ISSUE THREAD

**Issue by LalleSX** [ClearURLs/Addon#215]
# Adding URL rules to the Addon

When a site doesn't work as intended i all ways troubleshoot by disabling add-ons to se if they are the problem

But with clear-curls when i disable it it doesn't turn on after i leave the site like for example uBlock origin does.

I have had a few instances when clear-urls caused problems, (tho few) but i typically forget to turn it back on.

Hope it can be implemented! Love the addon :D

**Comment by marcstern** [comment]
This would definitely makes this add-on one of the most useful ones!
Currently, it's very difficult for me to use it as it breaks my own developments.
I hope you'll be able to implement this feature.
Thanks a lot.

**Comment by Acleacius** [comment]
Thanks for this amazing Addon!  :D
I have the same issue, for sites that I support like a game news site, giving me a link.
They need a coded url to get credit/percentage to help their site.
So please give this feature consideration, in the future.

I too, had to disable ClearURLs but then forgot it was off.  :(

**Comment by LeJared** [comment]
Yes, this is really much needed!
The add-on currently breaks some sites and the only way to handle this, is to turn it off and on again manually every time you visit such sites.

**Comment by beckerp86** [comment]
bump. It won't let you sort search results on Amazon.

**Comment by NoSkillPureAndy** [comment]
bumpitty. google search has an interesting new ai overview feature that is broken by clearurls.

**Comment by Te-M** [comment]
Still interested!

**Comment by davidnir** [comment]
One way CleanURLs breaks things in particular is that if you try to unsubscribe from an email sent by anyone using ngpvan.com, you get an error page. The one way around this I've found is to open the unsubscribe URL in a private window.

**Comment by marnovo** [comment]
Also breaks authentication of apps in browser. I have to remember to disable it every time. E.g. when using a native app that requires browser login via Google, etc.

**Comment by ImreLovasz** [comment]
Yeah, we really need to have a list to store URLs that should not be tempered with.

**Comment by inson1** [comment]
@LalleSX duplicate of #93

**Comment by Padde86** [comment]
This feature is definitely needed and I would very much appreciate it!

**Comment by OutlawCecil** [comment]
Agreed this is needed! AliExpress is broken by this extension currently.

**Comment by fabian-mv** [comment]
bump. some sites need this in order to work

**Comment by swampautomater** [comment]
bump. breaks aliexpress.

**Comment by garry-ut99** [comment]
Stop spamming and bumping duplicated thread clowns, you were already told in  https://github.com/ClearURLs/Addon/issues/215#issuecomment-1858511734 that it's a duplicate of:
- https://github.com/ClearURLs/Addon/issues/93

Go to the original thread, leave alone duplicates.

**Comment by Acleacius** [comment]
Never try to Clown the Clowns, Bozo!  😛 
Think of this thread more as a place we Friends gather and talk about new Clowning Techniques. 

Seriously, iirc there hasn't been an update to this Addon since it's release 4 years ago.
We are mostly a group of needy Clowns, hoping someone will have mercy on us and update this Addon.

Just remember, we are Clowns but your just a Gary! 😎 
https://www.tiktok.com/@gamespotdotcom/video/7358506125228018986
